src/Hamiltonians/HubbardMom1D.jl

```diff
@@ -1,18 +1,21 @@
 """
-    hubbard_dispersion(k)
-Dispersion relation for [`HubbardMom1D`](@ref). Returns `-2cos(k)`.
+    hubbard_dispersion(t,k)
+Dispersion relation for [`HubbardMom1D`](@ref). Returns ``-2(\\Re(t) \\cos(k) + \\Im(t) \\sin(k))``.
 
 See also [`continuum_dispersion`](@ref).
 """
-hubbard_dispersion(k) = -2cos(k)
+hubbard_dispersion(t::Real,k) = -2t.*cos(k)
+hubbard_dispersion(t::Complex,k) = -2*real(t).*cos(k) .- 2*imag(t).*sin(k)
+
 
 """
     continuum_dispersion(k)
-Dispersion relation for [`HubbardMom1D`](@ref). Returns `k^2`.
+Dispersion relation for [`HubbardMom1D`](@ref). Returns ``\\Re(t) k^2 - 2 \\Im(t) k``.
 
 See also [`hubbard_dispersion`](@ref).
 """
-continuum_dispersion(k) = k^2
+continuum_dispersion(t::Real,k) = t.*k^2
+continuum_dispersion(t::Complex,k) = real(t).*k^2 .- 2*imag(t).*k
```

Comment:
```suggestion
continuum_dispersion(t::Real, k) = t .* k^2
continuum_dispersion(t::Complex, k) = real(t) .* k^2 .- 2 * imag(t) .* k
```
Please try to follow these conventions
https://github.com/JuliaDiff/BlueStyle?tab=readme-ov-file#code-formatting